components/automate-ui/src/app/pages/infra-proxy/create-chef-server-modal/create-chef-server-modal.component.html

```diff
@@ -0,0 +1,47 @@
+<chef-modal [visible]="visible" (closeModal)="closeEvent()">
+  <h2 slot="title">Add Chef Server </h2>
+    <div class="flex-container">
+      <form [formGroup]="createForm">
+        <div class="margin">
+          <chef-form-field>
+            <label>
+              <span class="label">Name <span aria-hidden="true">*</span></span>
```

Comment:
- The Description should be a placeholder of Name.
- The Name should be treated as an ID(unique).

Ref: Team/Project Create Model popup.
![project-create-model](https://user-images.githubusercontent.com/44496937/70385770-4d4f3400-19b6-11ea-9624-7c91d273a663.png)